cmake_modules/optionDefaults.cmake

```diff
@@ -4,7 +4,12 @@ endif()
 
 if(NOT PREFIX)
     if(WIN32)
-        set(PREFIX "$ENV{ProgramFiles}")
+        set(prog86key "ProgramFiles(x86)")
+        if("$ENV{${prog86key}}" STREQUAL "")
```

Comment:
I agree I'll work on fixing that.  I need to close this down and put up the fixed branch that this was based off of (merge history got a little messy).  Reopening after closing.